### PR TITLE
Remove reverse relations from NetworkElementEntity to InspectionPort.

### PR DESCRIPTION
**Pull request by dmitryintel**

### CODE REVIEW

**Comment by arvindn05** [comment]
@dmitryintel branch out of date. also fix deprecation warnings on osgi test

File: src/main/java/org/osc/controller/nsc/entities/InspectionPortEntity.java [hunk 1]
```diff
@@ -62,14 +62,6 @@ public InspectionPortEntity(String elementId, NetworkElementEntity ingress, Netw
         this.ingressPort = ingress;
         this.egressPort = egress;
         this.inspectionHooks = new HashSet<>();
```
Comment (arvindn05) [inline src/main/java/org/osc/controller/nsc/entities/InspectionPortEntity.java]: > @JoinColumn(name = "egressId")

follow naming convention as egress_port_fk, ingress_port_fk etc

File: src/main/java/org/osc/controller/nsc/entities/InspectionHookEntity.java [hunk 2]
```diff
@@ -28,38 +28,43 @@
...
     @Id
     @GeneratedValue(generator = "uuid")
     @GenericGenerator(name = "uuid", strategy = "uuid2")
-    @Column(name = "hookId", unique = true)
+    @Column(name = "hook_id", unique = true)
     private String hookId;
 
     @OneToOne(cascade = CascadeType.ALL, orphanRemoval = false, fetch = EAGER, optional = true)
-    @JoinColumn(name = "inspectedPortId", nullable = true, updatable = true)
+    @JoinColumn(name = "inspected_port_id", nullable = true, updatable = true)
```
Comment (arvindn05) [inline src/main/java/org/osc/controller/nsc/entities/InspectionHookEntity.java]: add _fk to all foreign keys

File: src/main/java/org/osc/controller/nsc/entities/NetworkElementEntity.java [hunk 3]
```diff
@@ -30,43 +30,37 @@
...
     @Id
-    @Column(name = "elementId", unique = true)
+    @Column(name = "element_id", unique = true)
     private String elementId;
 
     // TODO : for SFC functionality
     @Transient
     private String parentId;
 
     @ElementCollection(fetch = EAGER)
```
Comment (arvindn05) [inline src/main/java/org/osc/controller/nsc/entities/NetworkElementEntity.java]: why is this eager and portIPs lazy. Should be both eager

Comment (dmitryintel) [reply src/main/java/org/osc/controller/nsc/entities/NetworkElementEntity.java]: Last time I tried, that resulted in org.hibernate.loader.MultipleBagFetchException

Comment (dmitryintel) [reply src/main/java/org/osc/controller/nsc/entities/NetworkElementEntity.java]: https://stackoverflow.com/questions/37282850/hibernate-org-hibernate-loader-multiplebagfetchexception-cannot-simultaneousl

http://blog.eyallupu.com/2010/06/hibernate-exception-simultaneously.html

Comment (dmitryintel) [reply src/main/java/org/osc/controller/nsc/entities/NetworkElementEntity.java]: Tried with sepearate entity for portIps and ManyToOne. Still the same issue.

Comment (dmitryintel) [reply src/main/java/org/osc/controller/nsc/entities/NetworkElementEntity.java]: OK. Lazy-loading mac IPs as discussed. But Using FetchType EAGER on both lists and @fetch(SELECT) on one of them also works.

**Pull request merged**